frontend/src/components/CuratorComponent.jsx

```diff
@@ -0,0 +1,343 @@
+import React from "react";
+import {
+  getRecommendationsFromPrompt,
+  getRecommendationsFromExistingTracks,
+} from "../api/recommendation";
+import { exportPlaylist } from "../api/playlist";
+import Loader from "./Loader";
+import HistoryDrawer from "./History/HistoryDrawer";
+import {
+  addExportedPlaylist,
+  addPrompt,
+  deletePrompt,
+  updatePromptSongs,
+} from "../api/history";
+
+import PromptView from "../pages/Curator/views/PromptView";
```

Comment:
I think we should change the file structure of these views since we moved the Curator to the components directory.

I think in `components` we should make a directory `Curator` and inside, have this file as `index.js` and then move the PromptView, ExportedView, and CuratedView to this directory as well.